web/index.js

```diff
@@ -1,403 +1,408 @@
-var uid = cuid();
-var hasher = crypto.createHash('sha256');
-hasher.update(os.hostname());
-var hostUid = hasher.digest('hex') + '-' + uid;
-server.method('getUid', function() { return hostUid });
-
-function checkHapiPluginError (name) {
-  return function (error) {
-    if (error) {
-      console.error('Failed loading a Hapi plugin: "' + name + '".');
-      throw error;
-    }
-  };
-}
-
-// Set up HAPI
-server.connection({
-  port: port,
-  // According to the HTTP spec and Chrome audit tool, Cache-Control headers should match what
-  // would be sent for 200 when a 304 (Not Modified) is sent.
-  routes: {
-    cache: { statuses: [200,304] },
-    cors: { origin: [ hostWithPort, 'https://changex.org', 'https://coderdojo.com', 'http://localhost'], credentials: true }
+if (process.env.NEW_RELIC_ENABLED === 'true') require('newrelic'); // eslint-disable-line global-require
+
+const env = process.env.NODE_ENV || 'development';
+
+const _ = require('lodash');
+const hapi = require('hapi');
+const blankie = require('blankie');
+const scooter = require('scooter');
+const chairo = require('chairo');
+const vision = require('vision');
+const inert = require('inert');
+const swagger = require('hapi-swagger');
+const etags = require('hapi-etags');
+const dust = require('hapi-dust');
+const path = require('path');
+const locale = require('locale');
+const crypto = require('crypto');
+const fs = require('fs-extra');
+const os = require('os');
+const blipp = require('blipp');
+const cuid = require('cuid');
+const hapiAuthCookie = require('hapi-auth-cookie');
+const frontend = require('cp-zen-frontend');
+const ip2country = require('hapi-ip2country-plugin');
+const good = require('good');
+const goodFile = require('good-file');
+const goodHTTP = require('good-http');
+const chairoCache = require('chairo-cache');
+const debug = require('debug')('cp-zen-platform:index');
+const languages = require('./config/languages.js');
+const options = require('./config/options.js');
+const controllers = require('./controllers');
+const httpError = require('./lib/http-error-handler');
+const preloader = require('./lib/plugins/seneca-preloader-dustjs');
+const leads = require('../lib/leads.js');
+const cdUsers = require('../lib/users.js');
+const agreements = require('../lib/agreements.js');
+const sys = require('../lib/sys.js');
+const configRoute = require('../lib/config.js');
+const oauth2 = require('../lib/oauth2.js');
+const profiles = require('../lib/profiles.js');
+const badges = require('../lib/badges.js');
+const events = require('../lib/events.js');
+const eventbrite = require('../lib/eventbrite.js');
+const polls = require('../lib/polls.js');
+const orgs = require('../lib/organisations.js');
+const dojos = require('../lib/dojos.js');
+const dustI18n = require('./lib/dust-i18n.js');
+const dustLoadJs = require('./lib/dust-loadjs.js');
+const dustLoadOG = require('./lib/dust-load-open-graph.js');
+
+exports.start = () => {
+  dustI18n();
```

Comment:
no need for those ?

Reply:
if you look at the files they were modified to be functions but i can revert that if need be